tracer/src/Datadog.Trace/Debugger/Snapshots/DebuggerSnapshotCreator.cs

```diff
@@ -672,11 +672,15 @@ internal string FinalizeMethodSnapshot<T>(string probeId, ref CaptureInfo<T> inf
 
         internal void FinalizeSnapshot(string methodName, string typeFullName, DateTimeOffset? startTime, string probeFilePath)
         {
+            var activeScope = Tracer.Instance.InternalActiveScope;
+            var traceId = activeScope?.Span?.TraceId.ToString();
+            var spanId = activeScope?.Span?.SpanId.ToString();
+
             AddStackInfo()
             .EndSnapshot(startTime)
             .EndDebugger()
             .AddLoggerInfo(methodName, typeFullName, probeFilePath)
-            .AddGeneralInfo(LiveDebugger.Instance?.ServiceName, null, null) // internal ticket ID 929
+            .AddGeneralInfo(LiveDebugger.Instance?.ServiceName, traceId, spanId) // internal ticket ID 929
```

Comment:
Do we still need `// internal ticket ID 929`?

Reply:
Done in f2e39f19694d2e9a44c777ae15e0fa38a3d73526.